contributor_docs/method.example.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * "This is a template for inline documentation of a method. Remove all text
+ * between double quotes for using this template. Some description about the
+ * method goes here. Explain in simple words, what the function does and what
+ * would be good/bad use cases for it. If there are any corners cases or warnings,
+ * do explain them over here."
+ *
+ * By default, the background is transparent.
+ *
+ * @method "methodName"
+ * @param {dataType} "variableName" "Description of the parameter"
+ * @param {dataType} ["variableName"] "Description of optional parameter"
+ * @chainable
+ *
+ * @example
+ * <div>
+ * <code>
+ * // "Comment explaining the example."
+ * "Your code goes here."
+ * </code>
+ * </div>
+ *
+ * @example
+ * <div>
+ * <code>
+ * // "Comment explaining the example."
+ * "Your code goes here"
+ * </code>
+ * </div>
+ *
+ * @alt
+ * "A single line precisely describing the first example"
+ * "A single line precisely describing the second example"
+ */
+
+/**
+ * @method "methodName"
+ * @param {"dataType"} "paramName" "Description of the param"
+ * @param {"dataType"} "paramName" "Description of the param"
+ * @chainable
+ */
+p5.prototype.methodName = function() {
+  // Comment explaining the code.
+  'Your code goes here';
+};
```

Comment:
what do you think about adding an additional example to this file that shows everything filled in? I think this might help to interpret the template

Reply:
Yes, that sounds good. For reference, i have already added a filled copy of the function as an image in the documentation. So i would add it, in the file also.